frontend/pages/interest/thank-you.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, Stack, Typography } from '@mui/material';
```

Comment:
Since there's a `/pages/interest` directory, we can move `/pages/interest.tsx` to `/pages/interest/index.tsx`